pnpm-lock.yaml

```diff
@@ -680,6 +680,15 @@ importers:
       '@auth/core':
         specifier: workspace:*
         version: link:../core
+      '@simplewebauthn/browser':
+        specifier: v9.0.0
+        version: 9.0.0
+      '@simplewebauthn/server':
+        specifier: v9.0.0
+        version: 9.0.0
```

Comment:
Should have not been added, `9.0.1` is already in the lockfile/core/next-auth libs. 

Make sure to add `@simplewebauthn/browser` and `@simplewebauthn/server` to dev dependencies with `^9.0.1`

Reply:
Fixed